src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json

```diff
@@ -362,38 +384,52 @@
       }
     }
   },
-  "component-height-50": {
-    "value": "20px",
+  "component-height-500": {
+    "value": "64px",
     "type": "sizing",
     "$extensions": {
       "spectrum-tokens": {
-        "uuid": "81ebf2b1-01fd-45a3-a099-72048e7d7991",
-        "name": "component-height-50",
+        "uuid": "b5211351-b7df-430a-93bf-906ac66ebf2e",
+        "name": "component-height-500",
         "constant-token-duplicate": false
       }
     }
   },
-  "component-height-500": {
-    "value": "64px",
+  "component-size-difference-down": {
+    "value": "2px",
     "type": "sizing",
     "$extensions": {
       "spectrum-tokens": {
-        "uuid": "b5211351-b7df-430a-93bf-906ac66ebf2e",
-        "name": "component-height-500",
+        "uuid": "",
+        "name": "component-size-difference-down",
         "constant-token-duplicate": false
       }
-    }
+    },
+    "description": "Down (2px)"
```

Comment:
Wondering where are these descriptions coming from?

Reply:
I included those based on the specs. I assumed we'd want to be specific about these values being down values. If it's redundant, I can remove and push another update to this PR tomorrow! 

<img width="999" alt="Screenshot 2024-01-08 at 6 18 52 PM" src="https://github.com/adobe/spectrum-tokens-studio-data/assets/129990790/e05e3c21-610d-452d-aa88-4a85607d98b4">

Reply:
Ah got it, thank you @nabuhasan! I got slightly confused since I do agree that without context, even the descriptions are not super clear. Agreed we should keep them for Tokens Studio – what if we added even more context? Something line `Down state size difference (2px)`, `Down state width ratio (3× component height)`, `Down state minimum component size (24 px)`

Reply:
@lynnhao I already went ahead and removed all the descriptions yesterday and Frank merged this PR. I do think it would be best to have a discussion about adding descriptions for everything at some point. Will make a note of your suggestion in our [non-color tokens paper doc](https://paper.dropbox.com/doc/Token-delivery-research-Oct-2023--CHXCI2nzXTcEr~20CztGpjgaAg-lsC6KQQRDyhjfHviO7CyR#:uid=726607802889538456186542&h2=Leveraging-descriptions-in-Tok)!